code/_helpers/global_lists_vr.dm

```diff
@@ -161,6 +161,177 @@ var/global/list/edible_trash = list(/obj/item/trash,
 				/obj/item/device/mmi/digital/posibrain,
 				/obj/item/device/aicard)
 
+var/global/list/cont_flavors = list(
+				"All" = cont_flavors_all,
+				"Acrid" = cont_flavors_acrid,
+				"Dirty" = cont_flavors_dirty,
+				"Musky" = cont_flavors_musky,
+				"Smelly" = cont_flavors_smelly,
+				"Wet" = cont_flavors_wet)
+
+var/global/list/cont_flavors_all = list("soggy",
```

Comment:
Generate this by concatenating all the lists together. Look down further in this file for all the places where lists are dynamically generated. Leave the list defined up here but an empty list, then populate it down there.

Opinion: Use the http://www.byond.com/docs/ref/info.html#/operator/|= to ensure that there are no duplicates. Opinion2: You'd better not hardcode the names of lists, use a for() over cont_flavors.

Reply:
I probably should've named the selection as general instead of all now that I think about it. Otherwise I'll need to exclude possible mode specific words in other lists from other collections (currently only "musky" but that's only because I didn't search for possible special additions for other lists yet)
How would I do the soft naming for the lists?

Reply:
If modes are made up of a few mode specific words 'then a bunch of other shared ones', then you should just have the modes be those specific words, and pick from a combination of 'general' and 'mode' instead of duplicating the shared words into each list.

Reply:
I suppose I could get rid of the generic selection and make the acrid one the default.